staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -3151,6 +3151,9 @@ func describeService(service *corev1.Service, endpointSlices []discoveryv1.Endpo
 		if service.Spec.InternalTrafficPolicy != nil {
 			w.Write(LEVEL_0, "Internal Traffic Policy:\t%s\n", *service.Spec.InternalTrafficPolicy)
 		}
+		if service.Spec.TrafficDistribution != nil {
```

Comment:
This field is located at the end of the `ServiceSpec` (https://github.com/kubernetes/kubernetes/blob/595516a14968ece55edfdc4945cd2e6a58183978/staging/src/k8s.io/api/core/v1/types.go#L5829). Would it be better to print it just before the `DescribeEvents` (and after `LoadBalancerSourceRanges`)?.

Reply:
Amended to move the line where requested.